java/core/src/main/java/co/worklytics/psoxy/impl/RESTApiSanitizerImpl.java

```diff
@@ -87,6 +94,11 @@ public RESTApiSanitizerImpl(@Assisted RESTRules rules,
     @Inject
     ParameterSchemaUtils parameterSchemaUtils;
 
+    @Inject @Named("ipEncryptionStrategy")
```

Comment:
I'd move `ipEncryptionStrategy` into a public property of `ReversibleTokenizationStrategy` (same for `ipHashStrategy` and `DeterministicTokenizationStrategy`), so we don't need to use them as hardcode values in modules and here.

Reply:
It's a instance of `ReversibleTokenizationStrategy` with different config than the regular one ... I think exposing all through `ReversibleTokenizationsStrategy` is too much. If anything I'd inject a factory-type thing with methods for default, ip, etc ... but the whole point of using a DI FW is to avoid factorie